loops/_voice_tracker.py

```
@@ -22,30 +25,28 @@ async def handleVoiceChange(user: discord.Member, before: discord.VoiceState, af
+            removeVoiceUser(member.id, member.guild.id)
     else:
-        updateVoiceUsers(active_members)
+        updateVoiceUsers([(m.id, m.guild.id) for m in active_members])
 
 
-def updateVoiceUsers(active_members: list[discord.Member]) -> None:
-    global voiceUsers
-    current_users_set = set(voiceUsers)
-    active_users_set = set(active_members)
+def updateVoiceUsers(active_user_ids: list[tuple[int, int]]) -> None:
+    global voice_user_ids
```

Flake8 (7.3.0)

[error] `global voice_user_ids` is unused: name is never assigned in scope

(F824)